### PR TITLE
weechat

### PR DESCRIPTION
**Pull request by jungle-boogie**
add weechat to irc

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -137,8 +137,9 @@ See also [Static Site Generators](#static-site-generators) and  [Content Managem
   * [Ostel](https://dev.guardianproject.info/projects/ostel/wiki/Server_Documentation) - Secure SIP telephony setup with ZRTP encryption `GPLv3` `Ruby`
 
 ### IRC
+  * [Convos](http://convos.by/) - Always online web IRC client ([Demo](http://demo.convos.by), [Source code](http://github.com/nordaaker/convos)) `ARTv2` `Perl`
   * [Shout](http://shout-irc.com/) -The self-hosted web IRC client ([Demo](http://demo.shout-irc.com/), [Source code](https://github.com/erming/shout)) `MIT` `Nodejs`
-  * [Convos](http://convos.by/) - Always online web IRC client ([Demo](http://demo.convos.by), [Source code](http://github.com/nordaaker/convos)) `ARTv2` `Perl` 
+  * [Weechat](https://weechat.org/) `GNUv3` `C`
```
Comment (nodiscc) [inline README.md]: missing description

**Comment by nodiscc** [comment]
Thank you, moved to https://github.com/Kickball/awesome-selfhosted/pull/133

**Comment by lorenzhs** [comment]
Would you be interested in adding https://github.com/glowing-bear/glowing-bear to the WeeChat section? It's a web frontend that connects to your weechat instance, a bit like irccloud.

**Comment by nodiscc** [comment]
@lorenzhs yes, please file a separate issue or send a Pull Request

**Pull request closed without merging**